tests/auth_tests/test_templates.py

```diff
@@ -52,3 +42,28 @@ def test_titles(self):
         response = PasswordChangeDoneView.as_view()(request)
         self.assertContains(response, '<title>Password change successful</title>')
         self.assertContains(response, '<h1>Password change successful</h1>')
+
+
+@override_settings(ROOT_URLCONF='auth_tests.urls')
+class AuthTemplateWithClientTests(TestCase):
+
+    def setUp(self):
+        self.pr_client = PasswordResetConfirmClient()
```

Comment:
After some more thinking, I don't think there's anything wrong with overriding `self.client`. In fact, I think it'll be less confusing when adding a new test ("oh, I forgot the use the special client."). If it causes confusion, we can rename it later, but for now, it'll keep the diff shorter also.